charts/fission-all/templates/_function-access-role.tpl

```diff
@@ -45,6 +45,24 @@ rules:
   - secrets
   verbs:
   - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: {{ .namespace }}
+  name: {{ .Release.Name }}-event-fetcher
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - "events"
+  verbs:
+  - "get"
+  - "list"
+  - "watch"
+  - "create"
+  - "update"
+  - "patch"  
```

Comment:
Please remove the following permission from fission-fetcher
```
- apiGroups:
  - ""
  resources:
  - pods
  verbs:
  - list
  ```
  
  And add pod get permission in the current event-fetcher role.
  
  Please rename event-fetcher role to fission-fetcher-websocket
  and same for rolebinding.

Reply:
With this, we will also change fetcher code WsStartHandler and WsEndHandler

Reply:
done